main.cpp

```diff
@@ -726,26 +764,27 @@ struct Filter
     double drive = 0.0; 
     double pianoIR;
     double signal;
-    std::string filterType = "Low Pass";
+    std::string Type = "Low Pass";
```

Comment:
this variable name did not need to be changed.
The new variable name does not conform to the course coding standard:
Variable names need to begin with a lower-case letter
- [ ] revise